src/NUnitFramework/benchmarks/nunit.framework.benchmarks/nunit.framework.benchmarks.csproj

```diff
@@ -19,4 +19,11 @@
       <ProjectReference Include="..\..\tests\nunit.framework.tests.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <PackageReference Update="Microsoft.CodeAnalysis.CSharp.CodeStyle">
+        <PrivateAssets>all</PrivateAssets>
+        <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      </PackageReference>
+    </ItemGroup>
+
```

Comment:
There is no need to add this at all as the package is already added in `Directory.Build.props`
Same hold for all other instances of this you added.

Reply:
For general purpose packages one could also use [Global Package References](https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management#global-package-references), which are automatically private assets.

Reply:
@manfred-brands Right you are.  I added them using the Nuget wizard, and didn't look closely there.  My bad.  Good you're awake :-)  

@lahma CPR is already there.